lib/xmlhttprequest.js

```diff
@@ -191,13 +208,13 @@ class XMLHttpRequest extends XMLHttpRequestEventTarget {
       method: 'get',
       responseHeaders: {},
       responseBuffers: [],
-      responseText: new String(),
+      responseText: '',
       responseType: '',
       requestHeaders: {
-        'X-Requested-With': 'XMLHttpRequest',
+        'X-Requested-With': 'XMLHttpRequest'
```

Comment:
Most exokit codebases use dangling comma.

Reply:
Ah, the eslint config changed it with `--fix`. I'll change whatever rule was causing that.

I personally prefer having the comma, too. :)